diff --git a/sepolicy/file.te b/sepolicy/file.te
index 2bf0cabd..e85766d8 100644
--- a/sepolicy/file.te
+++ b/sepolicy/file.te
@@ -11,5 +11,5 @@ type persist_property_file, file_type;
 type livedisplay_sysfs, sysfs_type, file_type;
 
 # Filesystems
-type exfat, sdcard_type, fs_type, mlstrustedobject;
-type ntfs, sdcard_type, fs_type, mlstrustedobject;
+#type exfat, sdcard_type, fs_type, mlstrustedobject;
+#type ntfs, sdcard_type, fs_type, mlstrustedobject;
diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index b5a4f0df..2b86e72a 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -6,9 +6,9 @@
 /data/local/userinit\.sh                u:object_r:userinit_data_exec:s0
 
 # For EXFAT/F2FS/NTFS partitions marked "formattable"
-/system/bin/mkfs\.exfat   u:object_r:mkfs_exec:s0
-/system/bin/mkfs\.f2fs    u:object_r:mkfs_exec:s0
-/system/bin/mkfs\.ntfs    u:object_r:mkfs_exec:s0
+#/system/bin/mkfs\.exfat   u:object_r:mkfs_exec:s0
+#/system/bin/mkfs\.f2fs    u:object_r:mkfs_exec:s0
+#/system/bin/mkfs\.ntfs    u:object_r:mkfs_exec:s0
 
 # For minivold in recovery
 /sbin/minivold            u:object_r:vold_exec:s0
@@ -42,8 +42,8 @@
 /sys/devices/virtual/graphics/fb0/color_enhance u:object_r:livedisplay_sysfs:s0
 
 # fsck
-/system/bin/fsck\.ntfs                          u:object_r:fsck_exec:s0
-/system/bin/fsck\.exfat                         u:object_r:fsck_exec:s0
+#/system/bin/fsck\.ntfs                          u:object_r:fsck_exec:s0
+#/system/bin/fsck\.exfat                         u:object_r:fsck_exec:s0
 
 # bash
 /system/xbin/bash                               u:object_r:shell_exec:s0
diff --git a/sepolicy/genfs_contexts b/sepolicy/genfs_contexts
index b5652a26..92228f62 100644
--- a/sepolicy/genfs_contexts
+++ b/sepolicy/genfs_contexts
@@ -1,3 +1,3 @@
-genfscon fuseblk / u:object_r:fuseblk:s0
-genfscon exfat / u:object_r:exfat:s0
-genfscon ntfs / u:object_r:ntfs:s0
+#genfscon fuseblk / u:object_r:fuseblk:s0
+#genfscon exfat / u:object_r:exfat:s0
+#genfscon ntfs / u:object_r:ntfs:s0
diff --git a/sepolicy/mkfs.te b/sepolicy/mkfs.te
index fe7c61bb..99489400 100644
--- a/sepolicy/mkfs.te
+++ b/sepolicy/mkfs.te
@@ -1,7 +1,7 @@
-type mkfs, domain;
-type mkfs_exec, exec_type, file_type;
+#type mkfs, domain;
+#type mkfs_exec, exec_type, file_type;
 
-init_daemon_domain(mkfs)
+#init_daemon_domain(mkfs)
 
 # Allow formatting userdata or cache partitions
 allow mkfs block_device:dir search;
diff --git a/sepolicy/service.te b/sepolicy/service.te
index f83efeda..d8b06640 100644
--- a/sepolicy/service.te
+++ b/sepolicy/service.te
@@ -1,4 +1,4 @@
-type edge_gesture_service, system_api_service, system_server_service, service_manager_type;
+#type edge_gesture_service, system_api_service, system_server_service, service_manager_type;
 type torch_service, system_api_service, system_server_service, service_manager_type;
 type kill_switch_service, system_api_service, system_server_service, service_manager_type;
 type cm_status_bar_service, system_api_service, system_server_service, service_manager_type;
diff --git a/sepolicy/service_contexts b/sepolicy/service_contexts
index 78b6a5cb..7a2bc838 100644
--- a/sepolicy/service_contexts
+++ b/sepolicy/service_contexts
@@ -1,4 +1,4 @@
-edgegestureservice                        u:object_r:edge_gesture_service:s0
+#edgegestureservice                        u:object_r:edge_gesture_service:s0
 torch                                     u:object_r:torch_service:s0
 killswitch                                u:object_r:kill_switch_service:s0
 cmstatusbar                               u:object_r:cm_status_bar_service:s0
